diff -rNu src.orig/satl1.f src.new/satl1.f
--- src.orig/satl1.f	2025-03-14 18:20:26.635345648 +0900
+++ src.new/satl1.f	2025-03-14 19:39:12.920497599 +0900
@@ -467,7 +467,7 @@
 	 GOTO 1400
       ENDIF
       NBULK=NST1-NSTEF
-      IF (NDTENS.NE.1.AND.NDTENS.NE.3) pause 'NDTENS must be 1 or 3'
+      IF (NDTENS.NE.1.AND.NDTENS.NE.3) STOP 11 !'NDTENS must be 1 or 3'
       READ (5,101) (LAFLAG(I), I=1,NST1)
       DO 348 I=1,NST1
 	 IF (LAFLAG(I).GT.NLAY) NLAY=LAFLAG(I)
diff -rNu src.orig/satl_sub.f src.new/satl_sub.f
--- src.orig/satl_sub.f	2025-03-14 18:20:26.631345635 +0900
+++ src.new/satl_sub.f	2025-03-14 19:36:41.544082848 +0900
@@ -7480,7 +7480,7 @@
         DO 11 J=1,N
           IF (CABS(A(I,J)).GT.CABS(AAMAX)) AAMAX=CABS(A(I,J))
 11      CONTINUE
-        IF (AAMAX.EQ.0.) PAUSE 'Singular matrix.'
+        IF (AAMAX.EQ.0.) STOP 21 !'Singular matrix.'
         VV(I)=1./AAMAX
 12    CONTINUE
       DO 19 J=1,N
@@ -9599,7 +9599,7 @@
       SUBROUTINE SPLINT(xa,ya,y2a,n,x,y,yp,klo,khi)
       dimension xa(n),ya(n),y2a(n)
       h=xa(khi)-xa(klo)
-      if (h.eq.0.) pause 'bad xa input. in SPLINT'
+      if (h.eq.0.) STOP 31 !'bad xa input. in SPLINT'
       a=(xa(khi)-x)/h
       b=(x-xa(klo))/h
       y=a*ya(klo)+b*ya(khi)+
@@ -10776,7 +10776,7 @@
           endif
         endif
 11    continue
-      pause 'brent exceed maximum iterations.'
+      STOP 32 !'brent exceed maximum iterations.'
 3     xmin=x
       brent=fx
       return
@@ -11728,7 +11728,7 @@
 C VOPT. For purpose of interpolation however we want it to be > than X(1)
             XMIN=XMIN+EINCR
             XVAL=XMIN
-            IF(XVAL.LT.X(1)) PAUSE 'AGAIN!!'
+            IF(XVAL.LT.X(1)) STOP 22 !'AGAIN!!'
             KLO=1
             KHI=2
 C this shift allows all points in the grid to be included in the energy 
@@ -11741,7 +11741,7 @@
 28          IF(XVAL.GE.X(KLO).AND.XVAL.LT.X(KHI))GOTO 27 
                   KLO=KLO+1
                   KHI=KHI+1
-                  IF(KHI.GT.NEM)PAUSE 'PROBLEM'
+                  IF(KHI.GT.NEM) STOP 23 !'PROBLEM'
                   GOTO 28
 27             CALL SPLINT(X,WORYT,WORYT2,NEM,XVAL,A(IB,I),
      %            AP(IB,I),KLO,KHI)
@@ -11875,7 +11875,7 @@
           Z(IP)=0.
 23      CONTINUE
 24    CONTINUE
-      PAUSE '50 iterations should never happen'
+      STOP 24 !'50 iterations should never happen'
       RETURN
       END
 C =======================================================================
